scripts/data_ingestion/download_statcast.py

```
@@ -0,0 +1,163 @@
+        df = pybaseball.batting_stats_bref(season)
+        
+        if df.empty:
+            print(f"  ⚠️  No batting data returned for {season}")
+            return False
+        
+        # Save to CSV
+        output_file = DATA_DIR / f"statcast_batting_{season}.csv"
+        df.to_csv(output_file, index=False)
+        
+        print(f"  ✅ Downloaded {len(df)} batting rows to {output_file}")
+        return True
```

Ruff (0.15.10)

[warning] Consider moving this statement to an `else` block

(TRY300)